config/crd/bases/postgresql.cnpg.io_clusters.yaml

```diff
@@ -1877,6 +1877,13 @@ spec:
               description:
                 description: Description of this PostgreSQL cluster
                 type: string
+              enablePDB:
```

Comment:
Please make sure that the new CRD field is also added into the older release (incl 1.18), as per the other discussion about 1.22 and 1.18 CRD compatibility. Thanks!
@mnencia